### DISCUSSION TITLE
support a more generic layerStyle

### DISCUSSION THREAD

**Discussion by tim-sheung** [chakra-ui/chakra-ui#8773] [Ideas]
### Description

Support a more generic layerStyle.

### Problem Statement/Justification

Currently `layerStyle` only supports

- Color or text color
- Background color
- Border width and border color
- Box shadow
- Opacity

There's a big limitation if we want more styles be used.

### Proposed Solution or API

Extend `layerStyle` to support the standard css styles.

**Comment by segunadebayo** [comment]
v3 `layerStyle` accepts any CSS. The old property whitelist is gone.
